guides/v2.2/frontend-dev-guide/layouts/xml-manage.md

```diff
@@ -383,7 +382,95 @@ In this file, reference the element having added the `remove` attribute:
 
 ## Replace elements {#layout_markup_replace_elements}
 
-To replace an element, [remove it] and add a new one.
+Elements can be replaced with custom blocks by one of the following methods below:
```

Comment:
I'm going to suggest something a little radical but think it is true to the position of Magento 2 code: remove "Replace Elements" section altogether. As far as I know it is not possible to do a true "replacement" because you would have to use a different name anyway – which, would not be truly replacing but instead removing one and adding another. I think it would be acceptable to have a _brief_ section about replacing the block class, but that is beyond the scope of layout XML anyway, because all that happens in `di.xml`. Example #1 here is still Magento 1, mostly, and I suggest removing it altogether. I haven't added any comments to Example #1, because I don't think it is salvageable.